chatAppTemplate_2/js/index.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  var clearResizeScroll, sendMessage, months;
+
+
+  conf = {
+    cursorcolor: "#696c75",
+    cursorwidth: "4px",
+    cursorborder: "none"
+  };
+
+  lol = {
+    cursorcolor: "#cdd2d6",
+    cursorwidth: "4px",
+    cursorborder: "none"
+  };
+
+  months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+
+  clearResizeScroll = function() {
+    $("#text").val("");
+    $(".messages").getNiceScroll(0).resize();
+    return $(".messages").getNiceScroll(0).doScrollTop(999999, 999);
+  };
+
+  sendMessage = function() {
```

Comment:
Ok so - this is where the message is sent? 

Have you managed to figure out a way to redirect where the message is sent?

Also looks like we are formatting the message here... lets include this information in our packet - but not do any formatting.

Lets say our packet looks like this for now:
```
{packet: {
  username: "name",
  message: "The Message",
  month: Month_value,
  year: year_value,
  hour: hour_value,
  minute: minute_value
  }
}